src/main/java/duke/task/TaskList.java

```diff
@@ -167,4 +168,25 @@ public void delete(int i) throws IndexOutOfBoundsException {
         }
         return output;
     }
+
+    /**
+     * Views task and schedule based on a specific date.
+     * @param date Date to be searched for in the form of DD/MM/YYYY
+     * @return An ArrayList of strings representing the list of tasks based on date provided
+     */
+    public ArrayList<Task> vfilter(LocalDate date) {
```

Comment:
Please rename the method to be more descriptive, such that it is easy to know what the method does just by reading the name.